encryption/caesar.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include "caesar.h"
+
+int euclideanModulo(int a, int b) {
+    return ((a) < 0 ? ((((a) % (b)) + (b)) % (b)) : ((a) % (b)));
+}
+
+namespace encryption {
+
+    std::string Caesar::encrypt(std::string userInput) {
+        for (char& ch: userInput) {
+            ch = static_cast<char>((ch + Caesar::shift - Caesar::ascii_offset)
+                % Caesar::ascii_modulo + Caesar::ascii_offset);
+        }
+        return userInput;
+    }
+
+    std::string Caesar::decrypt(std::string encryptedMessage) {
+        for (char& ch: encryptedMessage) {
+            ch = static_cast<char>((ch - Caesar::shift - Caesar::ascii_offset)
+                % Caesar::ascii_modulo + Caesar::ascii_offset);
+        }
+        return encryptedMessage;
+    }
+
+    std::string Caesar::improvedEncrypt(std::string userInput) {
+        srand(Caesar::seed);
```

Comment:
After changes, maybe let's think about init seed in scope of constructor and use it object?